components/feature/media/src/test/java/mozilla/components/feature/media/fullscreen/MediaSessionFullscreenFeatureTest.kt

```diff
@@ -254,4 +265,59 @@ class MediaSessionFullscreenFeatureTest {
 
         assertEquals(ActivityInfo.SCREEN_ORIENTATION_USER, activity.requestedOrientation)
     }
+
+    @Suppress("Deprecation")
+    @Test
+    @Config(sdk = [Build.VERSION_CODES.N])
+    fun `GIVEN the currently selected tab is in pip mode WHEN a custom tab loads THEN display custom tab in device's current orientation`() {
+        val activity = Robolectric.buildActivity(Activity::class.java).setup().get()
```

Comment:
Same as above, maybe we don't need a real instance of an activity?

Reply:
@Mugurell , I'm slowly becoming familiar with mock and RoboElectric. Just using mock is not working here probably it's just a stub. It seems that an activity object is needed to set the pip mode for the activity. Also MediaSessionFullScreenFeature needs to check the activity's orientation after pip mode is active.
Please correct me if I'm wrong

Reply:
Thank You ! Will try the above method next time